raphtory/src/core/tgraph/graph/tgraph.rs

```diff
@@ -4,29 +4,24 @@ use dashmap::DashMap;
 use rustc_hash::FxHasher;
 use serde::{Deserialize, Serialize};
 
-use crate::core::{
-    errors::{GraphError, MutateGraphError},
-    locked_view::LockedView,
-    storage::Entry,
-    time::TryIntoTime,
-    timeindex::TimeIndexOps,
-    tprop::TProp,
-    vertex::InputVertex,
-    vertex_ref::VertexRef,
-    Direction, Prop, PropUnwrap,
-};
-
-use super::{
-    edge::EdgeView,
-    edge_store::{EdgeLayer, EdgeStore},
-    graph_props::GraphProps,
-    node_store::NodeStore,
-    props::Meta,
-    tgraph_storage::{GraphStorage, LockedIter},
-    timer::{MaxCounter, MinCounter, TimeCounterTrait},
-    vertex::{ArcEdge, ArcVertex, Vertex},
-    EID, VID,
-};
+use crate::core::storage::locked_view::LockedView;
```

Comment:
do you prefer this style of imports?

Reply:
This is just from cargo fmt - can always change it

Reply:
I think it doesn't like the nested `{}`, I think you can still do `crate::core::{}` as long as you don't nest any imports with `{}` inside of it? Maybe?